source/concepts/networking/network-firewall.html.md.erb

```diff
@@ -0,0 +1,49 @@
+---
+owner_slack: "#modernisation-platform"
+title: AWS Network Firewall
+last_reviewed_on: 2022-02-06
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+We currently have setup in our core-network account AWS Network Firewall to monitor and control network traffic on a layer 3 level for transit gateway connection and cross account vpcs to protect reources
+such as EC2 instances, along with a general AWS resources
+
+## What is an AWS network firewall
+
+To learnmore about AWS firewall please follow the link bellow
+
+[AWS Firewall](https://aws.amazon.com/network-firewall/features/)
+
+## Amending Rules
+
+Rules can be amended by creating a pull request against the modernisation-platform repo and amending one of the following four file names from the core-network-services in the terrafrom environments folder.
+
+- [development_rules.json](https://github.com/ministryofjustice/modernisation-platform/blob/main/terraform/environments/core-network-services/development_rules.json)
+- [test_rules.json](https://github.com/ministryofjustice/modernisation-platform/blob/main/terraform/environments/core-network-services/test_rules.json)
+- [preproduction_rules.json](https://github.com/ministryofjustice/modernisation-platform/blob/main/terraform/environments/core-network-services/preproduction_rules.json)
+- [production_rules.json](https://github.com/ministryofjustice/modernisation-platform/blob/main/terraform/environments/core-network-services/production_rules.json)
+
+The rules files are broken down to environment and not account to make thing easier keep track of, to this end i have included a link to the MP cidr repo to help with forming new rules.
+
+[CIDR Ranges](https://github.com/ministryofjustice/modernisation-platform/blob/main/cidr-allocation.md)
+
+Rules are made up of a rule description an action which would ether be the DROP, PASS or ALERT action with would block, allow or monitor traffic based on the source ip or cidr range then a destination ip or cidr range, the port number is requird next finaily we have thr protocal used this can be ip or udp.
```

Comment:
```suggestion
Rules are made up of a rule description and an action which would either be the DROP, PASS or ALERT, which would block, allow or monitor traffic based on the source IP or CIDR range then a destination IP or CIDR range, the port number is required next finally the protocol used. This can be IP or UPD.
```

Reply:
Should this be TCP or UPD?